cmd/main.go

```diff
@@ -163,6 +170,7 @@ func main() {
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
 		Config: mgr.GetConfig(),
+		Dc:     dc,
```

Comment:
nit: Any specific reason for leaving this `Dc`, if I was in my IDE and typing out `r.` and looking at completions `DynamicClient` would be a much more descriptive "oh I need this" then `Dc`.